### PR TITLE
[WIP]Convert md links to href's in summaries

### PR DESCRIPTION
**Pull request by ChrisMaddock**
More towards dotnet/docs#1931.

### CODE REVIEW

File: xml/Microsoft.Build.BuildEngine/Project.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A <xref:Microsoft.Build.BuildEngine.Project> represents an [!INCLUDE[vstecmsbuild](~/includes/vstecmsbuild-md.md)] project. It is a container for items, properties and targets. It can load project content from in-memory XML or from an XML file, and can save to an XML file, preserving most white space and all XML comments.  
+ A <xref:Microsoft.Build.BuildEngine.Project> represents an MSBuild project. It is a container for items, properties and targets. It can load project content from in-memory XML or from an XML file, and can save to an XML file, preserving most white space and all XML comments.
```
Comment (rpetrusha) [inline xml/Microsoft.Build.BuildEngine/Project.xml]: Thanks for replacing the token!

File: xml/Microsoft.Build.Tasks.Windows/FileClassifier.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>FileClassifer</c> task. Use the <c>FileClassifer</c> element in your project file to create and execute this task. For usage and parameter information, see [FileClassifier](http://msdn.microsoft.com/library/14e03310-fcc0-4bb2-a84d-cda12be66367).</summary>
+    <summary>Implements the <c>FileClassifer</c> task. Use the <c>FileClassifer</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/fileclassifier-task">FileClassifier</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/FileClassifier.xml]: the title is "FileClassifier task"

File: xml/Microsoft.Build.Tasks.Windows/GenerateTemporaryTargetAssembly.xml [hunk 3]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>GenerateTemporaryTargetAssembly</c> task. Use the <c>GenerateTemporaryTargetAssembly</c> element in your project file to create and execute this task. For usage and parameter information, see [GenerateTemporaryTargetAssembly](/visualstudio/msbuild/generatetemporarytargetassembly-task).</summary>
+    <summary>Implements the <c>GenerateTemporaryTargetAssembly</c> task. Use the <c>GenerateTemporaryTargetAssembly</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/generatetemporarytargetassembly-task">GenerateTemporaryTargetAssembly</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/GenerateTemporaryTargetAssembly.xml]: the link target's title is "GenerateTemporaryTargetAssembly task"

File: xml/Microsoft.Build.Tasks.Windows/GetWinFXPath.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>GetWinFXPath</c> task. Use the <c>GetWinFXPath</c> element in your project file to create and execute this task. For usage and parameter information, see [GetWinFXPath](http://msdn.microsoft.com/library/b1dfb467-f3d3-47f3-83ef-af7b0e33a772).</summary>
+    <summary>Implements the <c>GetWinFXPath</c> task. Use the <c>GetWinFXPath</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/getwinfxpath-task">GetWinFXPath</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/GetWinFXPath.xml]: the title is "GetwinFXPath task"

File: xml/Microsoft.Build.Tasks.Windows/MarkupCompilePass2.xml [hunk 5]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>MarkupCompilePass2</c> task. Use the <c>MarkupCompilePass2</c> element in your project file to create and execute this task. For usage and parameter information, see [MarkupCompilePass2](http://msdn.microsoft.com/library/1d25689a-d21f-4b05-be26-95aa0ed4fd03).</summary>
+    <summary>Implements the <c>MarkupCompilePass2</c> task. Use the <c>MarkupCompilePass2</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/markupcompilepass2-task">MarkupCompilePass2</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/MarkupCompilePass2.xml]: title is "MarkupCompilePass2 task"

File: xml/Microsoft.Build.Tasks.Windows/MarkupCompilePass1.xml [hunk 6]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>MarkupCompilePass1</c> task. Use the <c>MarkupCompilePass1</c> element in your project file to create and execute this task. For usage and parameter information, see [MarkupCompilePass1](http://msdn.microsoft.com/library/693d6945-fd6f-4698-8f64-9dfcb71052d3).</summary>
+    <summary>Implements the <c>MarkupCompilePass1</c> task. Use the <c>MarkupCompilePass1</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/markupcompilepass1-task">MarkupCompilePass1</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/MarkupCompilePass1.xml]: title is "MarkupCompilePass2 task"

File: xml/Microsoft.Build.Tasks.Windows/MergeLocalizationDirectives.xml [hunk 7]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>MergeLocalizationDirectives</c> task. Use the <c>MergeLocalizationDirectives</c> element in your project file to create and execute this task. For usage and parameter information, see [MergeLocalizationDirectives](http://msdn.microsoft.com/library/9095b4f1-88da-4194-914b-ee1456826830).</summary>
+    <summary>Implements the <c>MergeLocalizationDirectives</c> task. Use the <c>MergeLocalizationDirectives</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/mergelocalizationdirectives-task">MergeLocalizationDirectives</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/MergeLocalizationDirectives.xml]: title is "MergeLocalizationDirectives task"

File: xml/Microsoft.Build.Tasks.Windows/ResourcesGenerator.xml [hunk 8]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>ResourcesGenerator</c> task. Use the <c>ResourcesGenerator</c> element in your project file to create and execute this task. For usage and parameter information, see [ResourcesGenerator](http://msdn.microsoft.com/library/e782bbac-9ee6-472b-8171-3ee008c77b4e).</summary>
+    <summary>Implements the <c>ResourcesGenerator</c> task. Use the <c>ResourcesGenerator</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/resourcesgenerator-task">ResourcesGenerator</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/ResourcesGenerator.xml]: title is "ResourcesGenerator task"

File: xml/Microsoft.Build.Tasks.Windows/UidManager.xml [hunk 9]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>UidManager</c> task. Use the <c>UidManager</c> element in your project file to create and execute this task. For usage and parameter information, see [UidManager](http://msdn.microsoft.com/library/4fc7b5a5-11b0-46ca-9656-8c2a0b08d1fe).</summary>
+    <summary>Implements the <c>UidManager</c> task. Use the <c>UidManager</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/uidmanager-task">UidManager</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/UidManager.xml]: title is "UidManager task"

File: xml/Microsoft.Build.Tasks.Windows/UpdateManifestForBrowserApplication.xml [hunk 10]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>UpdateManifestForBrowserApplication</c> task. Use the <c>UpdateManifestForBrowserApplication</c> element in your project file to create and execute this task. For usage and parameter information, see [UpdateManifestForBrowserApplication](http://msdn.microsoft.com/library/653339f7-654b-4d64-a26a-5c9f27036895).</summary>
+    <summary>Implements the <c>UpdateManifestForBrowserApplication</c> task. Use the <c>UpdateManifestForBrowserApplication</c> element in your project file to create and execute this task. For usage and parameter information, see <see href="https://docs.microsoft.com/visualstudio/msbuild/updatemanifestforbrowserapplication-task">UpdateManifestForBrowserApplication</see>.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Build.Tasks.Windows/UpdateManifestForBrowserApplication.xml]: title is "UpdateManifestForBrowserApplication task"

**Comment by ChrisMaddock** [comment]
Hi @rpetrusha - thanks for your feedback on this here. Afraid it looks like I wasn't able to come back to this one soon enough - there's been too many changes in master, and every file is now conflicting. Think it's best for me just to close this one - apologies for that.

**Pull request closed without merging**